fireeyeetp_connector.py

```diff
@@ -1492,7 +1465,7 @@ def main():
             login_url = FireeyeEtpConnector._get_phantom_base_url() + '/login'
 
             print("Accessing the Login page")
-            r = requests.get(login_url, verify=False)
+            r = requests.get(login_url, verify=False)  # nosemgrep
```

Comment:
We should be able to fix semgrep here and the one below.

Please see example at https://github.com/splunk-soar-connectors/forescoutcounteract/commit/2d63a3fd8c39c4e7d069e34022c08e5e85928820

Reply:
I guess, it's risky for the non-certified app as we are not sure that how much time API will take to send the response or how much data it will send.

So, we can think on this for certified apps but for the non-certified it's not safe.

Please suggest if you have anything else.

Reply:
Added timeout in main method but keeping it as it is in __make_rest_call_ method as per above comment.